gyro/lib/TaskManager.js

```diff
@@ -43,6 +43,8 @@ util.inherits(TaskManager, events.EventEmitter);
  * @param  {boolean} force force adding this task immediately
  */
 TaskManager.prototype.register = function(taskName, options, force) {
+    var i;
```

Comment:
is there a reason you're using `i` in both places, it seems potentially prone to breakage. It's probably worth just making these separate variables